### DISCUSSION TITLE
potentially hacked website of Adam Wathan (a blackmail e-mail from adam@adamwathan.me)

### DISCUSSION THREAD

**Discussion by SINOPW** [tailwindlabs/tailwindcss#15189] [General]
**I found a blackmail e-mail sent by adam@adamwathan.me to my business mailbox.**
![1](https://github.com/user-attachments/assets/ecbefd19-86fc-45f8-be7c-a8cf1890e9d6)

**When I copied "adamwathan.me" and entered into address bar, the following website which suggests the website belongs to Creator of Tailwind CSS showed up.**
![2](https://github.com/user-attachments/assets/5b9584ec-154a-48c5-9467-651fa4a76749)

**Comment by wongjn** [comment]
[Email spoofing](https://en.wikipedia.org/wiki/Email_spoofing) is fairly easily achieved, so it could be that. I'd also say that emails are not always on the same system as any website that share the same domain, so the website could be hacked but the emails are safe, or vice-versa.
